.eslintrc.js

```diff
@@ -1,17 +1,11 @@
 'use strict';
 
-const fs = require('fs');
-const path = require('path');
-const PACKAGE_NAME = require('./package').name;
-const SYMLINK_LOCATION = path.join(__dirname, 'node_modules', PACKAGE_NAME);
-
-// Symlink node_modules/{package name} to this directory so that ESLint resolves this plugin name correctly.
-if (!fs.existsSync(SYMLINK_LOCATION)) {
-  fs.symlinkSync(__dirname, SYMLINK_LOCATION);
-}
+// Register ourselves as a plugin to avoid `node_modules` trickery.
+const Plugins = require('eslint/lib/config/plugins');
+Plugins.define('prettier', require('.'));
```

Comment:
There's an accepted issue for it at https://github.com/eslint/eslint/issues/6237, but when I looked into it a little while ago it seemed like it could introduce problems even when absolute paths are used. Feel free to comment on that thread if you have any ideas for how to handle that, though.

Hmm, I hadn't realized modifying `node_modules` breaks yarn. Maybe we should keep `Plugins` somehow until we have a better way to refer to plugins by path.